packages/desktop/views/onboarding/views/create-from-ledger/create-from-ledger-route.enum.ts

```diff
@@ -1,4 +1,5 @@
 export enum CreateFromLedgerRoute {
     InstallLedger = 'installLedger',
     ConnectLedger = 'connectLedger',
+    ledgerConnectionGuide = 'ledgerConnectionGuide',
```

Comment:
Should be PascalCase:

```suggestion
    LedgerConnectionGuide = 'ledgerConnectionGuide',
```